Laravel Cron Upserts (#7279)
getsentry/sentry-docs: src/platform-includes/crons/setup/php.laravel.mdx

{
    $schedule->command('emails:send')
        ->everyHour()
        ->sentryMonitor(); // add this line
}
```

## Configure a Job Monitor

By default, the Laravel SDK will infer various parameters of your scheduled task.

> For more fine grained control, we do expose some optional parameters on the `sentryMonitor()` macro.

Reviewer: ```suggestion
For greater control, we expose some optional parameters on the `sentryMonitor()` macro.
```

```php {filename:app/Console/Kernel.php}
protected function schedule(Schedule $schedule)
{
    $schedule->command('emails:send')
        ->everyHour()
        ->sentryMonitor(
            // Specify the slug of the job monitor in case of duplicate commands or if the monitor was created in the UI
            monitorSlug: null,
            // Check-in margin in minutes
            checkInMargin: 5,
            // Max runtime in minutes
            maxRuntime: 15,

>             // In case you want to configure the job monitor exclusively in the UI, you can turn off sending the monitor config with the check-in.
>             // Passing a monitor-slug is required in this case.

Reviewer: ```suggestion
            // If you want to configure the job monitor exclusively in the UI, you can turn off sending the monitor config with the check-in.
            // Passing a monitorSlug is required in this case.
```

Reviewer: Reading this page, this comment didn't make sense to me - what does it mean to "turn off sending the monitor config with the check-in"?

Just wanted to flag in case it may be confusing to users too.

Author: By default, we sent a so-called monitor config along each check-in containing parameters such as the schedule and time zone. To allow users to opt out of this behaviour, we added `$updateMonitorConfig`.
